manifest.php

```diff
@@ -36,10 +36,10 @@
     'label' => 'LTI Proctoring',
     'description' => 'Grants access to the proctoring functionalities using LTI',
     'license' => 'GPL-2.0',
-    'version' => '2.1.0',
+    'version' => '2.1.1',
```

Comment:
could you at least update the minor number. Here we might have some deployment taking 2.1.1 from ~2.1.0 and not taking the taoLti version.
With a minor change (instead of a fix) we will have to explicitly update the composer.json, it reduces the chance of issue